packages/handleRichTextBoxKeyEventNavigation/handleRichTextBoxKeyEventNavigation.js

```diff
@@ -44,3 +73,67 @@ export default function(e, record) {
     }
     return false;
 }
+
+// Depends on the implementation, there might be nested div elements
+// for a record's rich text box DOMNode. However, record.focus()
+// will only focus on one of them. When tabbing to the previous focusable
+// element, we need to skip all the elements for record's DOMNode.
+function findPrevFocusableElem_(record) {
+    const tabbableElems = [
+        ...document.querySelectorAll(TABBABLE_ELEMENT_SELECTOR),
+    ];
+    const recordDom = record.getDom();
+    if (!recordDom) {
+        return null;
+    }
+    const recordElems = [
+        ...recordDom.querySelectorAll(TABBABLE_ELEMENT_SELECTOR),
+    ];
+    for (let i = 0; i < tabbableElems.length; i++) {
+        if (tabbableElems[i] === recordDom) {
+            if (i > 0) {
+                return tabbableElems[i - 1];
+            }
+        }
+        if (recordElems.includes(tabbableElems[i])) {
+            const index = tabbableElems.findIndex(
+                item => item === recordElems[0]
+            );
+            if (index > 0) {
+                return tabbableElems[index - 1];
+            }
+        }
+    }
+
+    return null;
+}
+
+function findNextFocusableElem_(record) {
```

Comment:
this function's got lots of duplicate code shared with the previous function, wondering if we can refactor any of it out...